src/sass/partials/_colours.scss

```diff
@@ -1,37 +1,40 @@
-$white: #fff;
-$black: #333;
-$darkest-black: #000;
-$dark-blue: #568B9A;
-$red: #C52253;
-$grey: #f2f2f2;
-$grey-white: #f4f4f4;
-$dark-grey: #949494;
-$green: #3DA167;
-$pale-green: #7BC3A3;
-$grey-black: #777;
-
-$salmon-pink: #fe8359;
-$pop-pink: #e42e3d;
-$pop-blue: #88DAE9;
-
-$pop-purple: #ffaf9c;
-$orange: #fe6633;
-$pale-orange:#FF8962;
-$pop-yellow: #fecc33;
-$alt-pink: #fe3366;
-
-$nav-link-color: $darkest-black;
-$nav-link-visited-color: $nav-link-color;
-
-$volunter-type-bg-color-1: rgba(236, 86, 112, 0.6);
-$volunter-name-bg-color-1: rgba(236, 86, 112, 0.6);
-
-$volunter-type-bg-color-2: rgba(247, 203, 255, 0.6);
-$volunter-name-bg-color-2: rgba(247, 203, 255, 0.6);
-
-$grey-color:       #828282;
-$grey-color-light: lighten($grey-color, 30%);
-$grey-color-dark:  darken($grey-color, 25%);
-
-$brand-color:      #ff6525;
-$link-hover:	   darken($brand-color, 15%);
\ No newline at end of file
+$white: 						#fff;
```

Comment:
I'll clean this up. For some reason it's all over the place on here but not on my Sublime.